### PR TITLE
[WIP] Landing Page Redesign

### PR DESCRIPTION
**Pull request by Annastacia-dev**
## What

Redesign some sections of the landing page.
Refactor code for some sections of the landing page.
Restyle( increase some margins and spacings)

## Screenshots
Screenshot from 2024-05-16 13-33-54
Screenshot from 2024-05-16 13-33-24

### CODE REVIEW

**Comment by JudahSan** [comment]
Props for improving the design @Annastacia-dev 
Kindly fix the conflicting files

**Comment by Annastacia-dev** [comment]
> Props for improving the design @Annastacia-dev Kindly fix the conflicting files

Hello, the PR is still a work in progress will fix and also let you know when it's ready for review, let's hold on that

**Comment by Annastacia-dev** [comment]
@JudahSan the PR is ready

File: app/views/layouts/_footer.html.erb
Comment (JudahSan) [inline app/views/layouts/_footer.html.erb:1]: image

Footer comparison between the old and new design.

File: app/views/layouts/_navbar.html.erb
Comment (JudahSan) [inline app/views/layouts/_navbar.html.erb:1]: image

@Annastacia-dev  Missing closing navbar tag: `</navbar>`

File: app/views/layouts/_footer.html.erb
Comment (JudahSan) [reply app/views/layouts/_footer.html.erb:1]: @Annastacia-dev The footer is great on big screens but lacks optimization for small screens.

File: app/views/layouts/_navbar.html.erb
Comment (JudahSan) [inline app/views/layouts/_navbar.html.erb:1]: image

@Annastacia-dev  Missing closing navbar tag: `</navbar>`

**Review by JudahSan — CHANGES_REQUESTED** [review]

**Comment by JudahSan** [comment]
> @JudahSan the PR is ready for review

Reviewed. Nice work.

**Comment by denissellu** [comment]
@Annastacia-dev this is great work! thanks!

I love that you are bring in annotate! 👏

how would you feel to cherry-pick it in to another PR? this commit? https://github.com/nairuby/arc_platform/pull/84/commits/63a113f311d1f495e3ec8dcd2e83a952af9445f2
it'll help make review easier and keep this PR focused on the style changes

File: app/views/layouts/_navbar.html.erb
Comment (Annastacia-dev) [reply app/views/layouts/_navbar.html.erb:1]: noted, will fix

File: app/views/landing/home/_activities.html.erb [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
+  {
+    image: 'activities_local_workshops.png',
+    title: 'workshops'
+  },
+  {
+    image: 'activities_local_hackathons.png',
+    title: 'hackathons'
+  }
+] %>
+
```
Comment (denissellu) [inline app/views/landing/home/_activities.html.erb:23]: please move this in to a helper

File: app/views/landing/home/_chapters.html.erb [hunk 2]
```diff
@@ -1,59 +1,46 @@
...
+    image: 'country_tanzania.png',
+    alt: 'ARC_Tanzania'
+  },
+  {
+    country: 'Uganda',
+    image: 'country_uganda.png',
+    alt: 'ARC_Uganda'
+  }
+
+] %>
```
Comment (denissellu) [inline app/views/landing/home/_chapters.html.erb:23]: move this too in to a helper 🙏

File: app/views/landing/home/_featured_sponsors.html.erb [hunk 3]
```diff
@@ -1,17 +1,27 @@
...
+    link: 'https://www.nairobits.com/',
+    alt:'Nairobits'
+  },
+  {
+    image: 'sponsors/current/finplus.png',
+    link: 'https://finplusgroup.com',
+    alt: 'Finplus Group'
+  }
+
+] %>
```
Comment (denissellu) [inline app/views/landing/home/_featured_sponsors.html.erb:13]: in to a helper too please

File: app/views/landing/home/_previous_sponsors.html.erb [hunk 4]
```diff
@@ -1,70 +1,64 @@
...
-      <% end %>
-    </div>
-
-    <div class="place-self-center w-20 card">
-      <%= link_to 'https://www.turing.com/', target: '_blank' do %>
-        <%= image_tag 'sponsors/previous/turing.png', alt: 'Turing',
-                      class: 'max-w-none w-40 md:w-20 pb-6' %>
-      <% end %>
-    </div>
+] %>
```
Comment (denissellu) [inline app/views/landing/home/_previous_sponsors.html.erb:33]: into a helper 🙏

File: app/views/layouts/_footer.html.erb [hunk 5]
```diff
@@ -1,122 +1,100 @@
...
+    image: "brands_linkedin.png",
+    show: true
+  },
+  {
+    alt: "ARC Github",
+    link: "https://github.com/nairuby",
+    image: "brands_github.png",
+    show: true
+  }
+] %>
```
Comment (denissellu) [inline app/views/layouts/_footer.html.erb:38]: into a helper 🙏

**Review by denissellu — CHANGES_REQUESTED** [review]
@Annastacia-dev thanks for the work on here!

just small changes to move the arrays create in the partials into helpers, so the templates can be tidier and easier to manage. 

what do you think?

**Review by JudahSan — APPROVED** [review]

**Review by denissellu — APPROVED** [review]
👏 nice one!

**Pull request merged**